internal/openfga/stores.go

```diff
@@ -279,7 +278,7 @@ func (s *OpenFGAStore) ListPermissions(ctx context.Context, ID string, continuat
 		eMsg = fmt.Sprintf("%s%v - %s\n", eMsg, n, e.Error())
 	}
 
-	return permissions, tMap, fmt.Errorf(eMsg)
+	return permissions, tMap, fmt.Errorf("%s", eMsg)
```

Comment:
nit: I think you can use `errors.New`

Reply:
Yes thank you, actually that's what I wanted to use but Ale in the past has asked me to stick with fmt.Errorf.
I would like to change it to errors.New, do you approve @shipperizer ?

Reply:
I pushed the change as a new commit in the meantime

Reply:
I'm fine either way

Reply:
BTW I just realized I didn't explain why I had to do this, I only mentioned the "vet command complaints".
Basically right now with go1.24 if you use `fmt.Errorf` with a string var it complains and the build fails.
It has to be a string constant (which in Go is actually a constant string literal).
I'll add this explanation to the description for context.

Reply:
right, my preference would be sticking with fmt.Errorf
sorry I read the pr description before the change and only came round to it now

anyway it is such a trivial thing that if you guys are happy with errors then let's stick to it

dunno if there is anything on the canonical golang style guide, but happy to go with whatever the team prefers